monai/networks/nets/vitautoenc.py

```diff
@@ -74,6 +74,9 @@ def __init__(
         if hidden_size % num_heads != 0:
             raise ValueError("hidden_size should be divisible by num_heads.")
 
+        if spatial_dims == 2:
```

Comment:
@wyli @finalelement , are we going to support 2D later? If not, I think we don't need this confusing arg?
And if following MONAI block & layer factories, it may be not hard to support both 2D and 3D, like current BasicUNet or DenseNet, etc? @yiheng-wang-nv please feel free to correct me if I misunderstood it.

Thanks.

Reply:
yes, this will be extended to support 2D, but it'll need some testings. For now it just raises an error...